test/k6/src/tests/instances.js

```diff
@@ -73,12 +77,17 @@ export function setup() {
 
 // TC01 - POST instance
 function TC01_PostInstance(data) {
+  const options = {
```

Comment:
Would reconsider including the options here. Look into how partyId and appId is set for the instance before sending to storage. Could personNumber be handled the same way? Or should we rather include all the properties in options? Using both approaches seem a bit messy.

Reply:
I would go for putting all the properties in options as it seems easier to handle validating specific parameters. Plus, option represents both `personNumber` and `orgNumber` which seemed a bit easy to handle than passing two parameters when calling this function. Hereby, my money is on passing the parameters as an object rather separate separate parameters. 🤔